src/components/landing/pages/history/common/entry-menu/entry-menu.tsx

```diff
@@ -40,17 +44,11 @@ const EntryMenu: React.FC<EntryMenuProps> = ({ id, location, isDark, onRemove, o
             <Trans i18nKey="landing.history.menu.entryRemote"/>
           </Dropdown.Item>
         </ShowIf>
-        <Dropdown.Item onClick={onRemove}>
-          <ForkAwesomeIcon icon="archive" fixedWidth={true} className="mx-2"/>
-          <Trans i18nKey="landing.history.menu.removeEntry"/>
-        </Dropdown.Item>
+        <RemoveNoteEntryItem onRemove={onRemove} />
 
         <Dropdown.Divider/>
 
-        <Dropdown.Item onClick={onDelete}>
-          <ForkAwesomeIcon icon="trash" fixedWidth={true} className="mx-2"/>
-          <Trans i18nKey="landing.history.menu.deleteNote"/>
-        </Dropdown.Item>
+        <DeleteNoteItem onDelete={onDelete} />
```

Comment:
Aren't those the basically the same, but differ only in the i18nKeys, the callback and the icon?

Reply:
You're right with that. I thought about even more abstraction too, but didn't do that as the exact `DeleteNoteItem` component can be used in the dropdown menu of the editor. Furthermore, I'm unsure if we want to have a component with that many attributes (because each i18nKey would need to be an attribute with higher abstraction).

Reply:
Since you wrote it, you're the expert on that topic and I trust your instincts there. It was just something I noticed while reviewing this

Reply:
What about putting them together into one component that has an option `variant = 'deleteNote' | 'removeHistoryItem'` ? The component would then use the right icon and i18nKeys by switching them internally.

Reply:
Done that as described in the answer above.